xchange-coinbase/src/main/java/org/knowm/xchange/coinbase/v2/CoinbaseV2Digest.java

```diff
@@ -24,7 +24,9 @@ public String digestParams(RestInvocation restInvocation) {
         restInvocation.getParamValue(HeaderParam.class, CB_ACCESS_TIMESTAMP).toString();
     String methodPath =
         "/v2" + restInvocation.getPath(); // todo  - move the /v2 bit of the path to the interface
-    String message = timestamp + restInvocation.getHttpMethod() + methodPath;
+    final String queryString = restInvocation.getQueryString();
+    final String queryStringWithSeparator = queryString.isEmpty() ? "" : "?" + queryString;
+    String message = timestamp + restInvocation.getHttpMethod() + methodPath + queryStringWithSeparator;
     String body = null; // todo - deal with POST requests
 
     String sign = DigestUtils.bytesToHex(getMac().doFinal(message.getBytes()));
```

Comment:
can you cleanup comments and `showCurl` method since its not in use.

also return `DigestUtils.bytesToHex(getMac().doFinal(message.getBytes()))` inline